tox.ini

```diff
@@ -87,10 +88,10 @@ commands =
 commands =
     bash -c "if [ ! -d "{toxinidir}/tests/common" ]; then \
                git clone https://git.openstack.org/openstack/openstack-ansible-tests {toxinidir}/tests/common; \
-               pushd {toxinidir}/tests/common; \
-                 git checkout {env:OSA_TEST_RELEASE:master}; \
-               popd; \
-             fi"
+             fi; \
+             pushd {toxinidir}/tests/common; \
+               git checkout {env:OSA_TEST_RELEASE:master}; \
+             popd"
```

Comment:
This ordering won't update from master if the dir exists (meaning stale code), a fetch should be done beforehand as well.

`git fetch origin {env:OSA_TEST_RELEASE:master}`

Reply:
Yeah, you're right, I need to rewrite this.